Feature/verificar_sugestoes

Lozavival opened the pull request:
Implementa a verificação e validação de sugestões de empresas enviadas por usuários.

Closes #41

Lozavival committed:
feat(company/suggestion): add status field to suggestion and make migrations

Lozavival committed:
feat(moderation): create company suggestion validation

Create pending suggestions list page, suggestion validation form page and add link to suggestions list to profile page.

Lozavival committed:
feat(moderation): give company permissions to moderator group

Lozavival committed:
test(moderation): add tests for company suggestion validation

ra247362 reviewed (changes requested):
Há alguns problemas, notadamente a falta da possiblidade de mudar o nome da empresa submetida pelo usuário e a falta de uma imagem padrão, o que vai quebrar o site se houver engano por parte do moderador. O código em si parece bom.

Lozavival committed:
feat(moderation): add name to validation form and make all fields required

Add 'name' field to suggestion validation form, make all fields required and add/change tests.

Lozavival committed:
fix(moderation): fix bug that threw exception when form was invalid

ra247362 reviewed (approved):
Tudo certo. Aprovado!
